OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -66,8 +85,24 @@ SEARCH_APP_USER_PASSWORD=<Search User Password>
 Run the following command to create the Admin Server container:
 
 ```
-$ docker run -i -t --name $ADMIN_SERVER_CONTAINER_NAME --network=WCPortalNET -p <Any Free Port>:$ADMIN_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName
+$ docker run -i -t --name <ADMIN_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:<ADMIN_PORT> -v <DATA_VOLUME>:/u01/oracle/user_projects --env-file <webcenter.env.list> oracle/wcportal:12.2.1.4
 ```
+Sample Run Command
+```
+$ docker run -i -t --name WCPAdminContainer --network=WCPortalNET -p 7001:7001 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4
+
+```
+Admin Container start up command explained:
+
+| 		Parameter    	 |     Parameter Name             | 							     		Description			                               |
+| :--------------------: | :----------------------------: | :---------------------------------------------------------------------------------------:  |
+| --name                 | ADMIN_CONTAINER_NAME                 |  Set to Admin Server Container Name                                                        |
+| --network              | NETWORK_NAME                   | User-defined network to connect to; use the one created earlier ‘WCPortalNET’.             |
+| -p                     | HostFreePort:ADMIN_PORT        | WebLogic Admin server port; Maps the container port to host's port.                      |
+| -v                    | DATA_VOLUME                    | Mounts the host directory as a Volume.                                                     |
+| --env-file             | webcenter.env.list|  `webcenter.env.list` sets the environment variables.                                    |
+| oracle/wcportal:12.2.1.4   | REPOSITORY:TAG           | Repository name:Tag name of the image created using buildDockerImage.sh                                                           |
```

Comment:
"tag" should be lowercase, and it doesn't need to be created by `buildDockerImage.sh`. I would just state, "The (optional) repo, name and tag of the image"